### ISSUE TITLE
[Bug]: [BAD_HTTP_STATUS: 403] Potential causes: IP block or streaming URL deciphering failed

### ISSUE THREAD

**Issue by GatoAmarilloBicolor** [FreeTubeApp/FreeTube#6601]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.

### Describe the bug

1. Just play two or tree videos
2. then the next video just play 2 seconds and give me that message: [BAD_HTTP_STATUS: 403] Potential causes: IP block or streaming URL deciphering failed

### Expected Behavior

Youtube are working fine in the browser, this messages are always even if i try to mask with indivious.

### Issue Labels

accessibility issue

### FreeTube Version

0.22.1

### Operating System Version

Linux Fedora

### Installation Method

.rpm

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by absidue** [comment]
Duplicate of #6385
